### PR TITLE
[instant] Don't allow changing of assets when order in progress

### PR DESCRIPTION
**Pull request by steveklebanoff**
## Description

Don't allow user to change assets when an order is happening.

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Comment by steveklebanoff** [comment]
Thinking about it more, I wonder if we should keep the asset text and icon clickable for now - as it's a way to restart the process.  The code wouldn't be as clean though, as there's no longer a one-to-one mapping, it's nice right now to be able to say "Disabled input means no token selection"

**Comment by steveklebanoff** [comment]
I updated the code to re-enable the token selection once the order is successful, even though the code isn't as clean and it's a non-obvious UX interaction, I think it's good to have _some way_ to restart the process on success right now.

File: packages/instant/src/components/erc20_asset_amount_input.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface ERC20AssetAmountInputProps {
     startingFontSizePx: number;
     fontColor?: ColorOption;
     isDisabled: boolean;
```
Comment (BMillman19) [inline packages/instant/src/components/erc20_asset_amount_input.tsx]: Lets rename this prop into something more specific like `isInputDisabled`

Comment (steveklebanoff) [reply packages/instant/src/components/erc20_asset_amount_input.tsx]: Done in b7eb2e8

**Pull request merged**